src/core/react/hooks/useKeyboardListNavigation.tsx

```diff
@@ -1,16 +1,26 @@
 import { RefObject, SetStateAction, useEffect, useState } from 'react';
 
-import { DOWN_KEY_CODE, ENTER_KEY_CODE, UP_KEY_CODE } from 'LumX/core/constants';
+import { BACKSPACE_KEY_CODE, DOWN_KEY_CODE, ENTER_KEY_CODE, UP_KEY_CODE } from 'LumX/core/constants';
+
+import get from 'lodash/get';
 
 /////////////////////////////
 
 type useKeyboardListNavigationType = (
```

Comment:
Unfortunately we need the type [here](https://github.com/lumapps/design-system/blob/master/src/components/list/react/List.tsx#L71). Can't think of a way to avoid exporting that type, do you think of any ?